### DISCUSSION TITLE
When applying for the Student Developer Pack I cant use my phone's back camera??

### DISCUSSION THREAD

**Discussion by ethanallenn** [community/community#118274] [GitHub Education]
### Hi everyone,

The front camera works fine but I'm not able to select my back camera to take a picture of my student portal. I previously was accepted by using this method around a year ago when I was able to use my back camera. The portal has all the data I need to get approval but even when I use my front camera and turn my phone around the image is mirrored. 
<img width="328" alt="Screenshot 2024-04-08 at 10 10 49" src="https://github.com/community/community/assets/86062422/ff102ece-ba63-4942-b6e3-9c4fe809165a">

**Comment by ethanallenn** [comment]
Last time I was accepted I used the student portal? 
<img width="323" alt="Screenshot 2024-04-08 at 10 13 07" src="https://github.com/community/community/assets/86062422/ea3b55fa-57d0-4ad4-bc91-a73183d0a8e5">

**Comment by samus-aran** [comment]
Hi @ethanallenn 👋 

I suspect the issue wasn't the front camera but ensuring you have got the school name on your evidence. Try again with a piece of evidence that has all these details that the rejection reasons are stating. 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
